### DISCUSSION TITLE
基金亏，丢骰子也亏，亏麻了啊

### DISCUSSION THREAD

**Discussion by GuoJiaxing1997** [BurnHair/docs#145] [分享讨论]
![image](https://github.com/BurnHair/docs/assets/88412036/a72da678-c404-4d2d-b348-5d5dfa4007a5)

这还是在收益期望为+0.5的前提上，都能亏成这样，所以赌博岂不是亏得底裤都不剩了

**Comment by BennyThink** [comment]
加油，你再多丢几天，因为丢骰子的功能马上就要下线了。

**Reply by GuoJiaxing1997** [reply]
泪目，还是很喜欢这个功能的
